CI/azure-pipelines-CI.yaml

```
@@ -0,0 +1,40 @@
+---
+trigger:
+  branches:
+    include:
+      - 'feature/*'
+  tags:
+    exclude: ['*']
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+variables:
+  protoBufFile: 'protoc-3.6.1-linux-x86_64.zip'
+  protoBufEnvPath:  '$(system.defaultWorkingDirectory)/protoc'
+  protobufWGet: 'https://github.com/protocolbuffers/protobuf/releases/download/v3.6.1/protoc-3.6.1-linux-x86_64.zip'
+
+steps:
+  - task: DotNetCoreCLI@2
+    displayName: 'dotnet Restore'
+    inputs:
+      command: 'restore'
+      projects: 'Emif.Adapter.Schema.csproj'
+      feedsToUse: config
+      nugetConfigPath: NuGet.config
+  - bash: |
+     wget $(protobufWGet)
+     unzip $(protoBufFile) -d $(protoBufEnvPath)
+     export PATH=$PATH:$(protoBufEnvPath)/bin
+    displayName: 'Get ProtoBuf compiler'
+  - bash: |
+     protoc --proto_path=protobuf/emif/adapter/schema --csharp_out=Emif/Adapter/Schema --csharp_opt=base_namespace=Emif.Adapter.Schema acknowledgements.proto
+     protoc --proto_path=protobuf/emif/adapter/schema --csharp_out=Emif/Adapter/Schema --csharp_opt=base_namespace=Emif.Adapter.Schema primitives.proto
+     protoc --proto_path=protobuf/emif/adapter/schema --csharp_out=Emif/Adapter/Schema --csharp_opt=base_namespace=Emif.Adapter.Schema reminders.proto
+     protoc --proto_path=protobuf/emif/adapter/schema --csharp_out=Emif/Adapter/Schema --csharp_opt=base_namespace=Emif.Adapter.Schema submission_status.proto
+     protoc --proto_path=protobuf/emif/adapter/schema --csharp_out=Emif/Adapter/Schema --csharp_opt=base_namespace=Emif.Adapter.Schema volumes.proto
+    displayName: 'Generate C# files per proto'
```

```suggestion
     protoc ./protobuf/emif/adapter/schema/*.proto --csharp_out=Emif/Adapter/Schema --proto_path=./protobuf
    displayName: 'Generate C# files'
```
Ellers tror jeg at namespace settes automatisk riktig basert på pakkenavn